client/components/gsuite/gsuite-new-user-list/style.scss

```diff
@@ -28,6 +28,14 @@
 	@include breakpoint-deprecated( '>480px' ) {
 		width: auto;
 	}
+
+	@include breakpoint-deprecated( '>660px' ) {
+		margin-bottom: 20px;
+	}
+
+	@include breakpoint-deprecated( '>800px' ) {
+		margin-bottom: 0;
+	}
```

Comment:
This fixes a missing margin between the two last button of the Google Workspace form:

Before | After
------ | -----
![before](https://user-images.githubusercontent.com/594356/117173548-2bd1c200-adcd-11eb-92d8-4879ecff8591.png) | ![after](https://user-images.githubusercontent.com/594356/117173567-2eccb280-adcd-11eb-9d0d-22be6231a787.png)

stephanethomas55